core/src/main/java/io/questdb/griffin/SqlKeywords.java

```diff
@@ -1980,6 +1980,25 @@ public static boolean isSemicolon(CharSequence token) {
         return Chars.equals(token, ';');
     }
 
+    public static boolean isServerConfKeyword(CharSequence tok) {
+        if (tok.length() != 11) {
+            return false;
+        }
+
+        int i = 0;
+        return (tok.charAt(i++) | 32) == 's'
```

Comment:
nit: other dbs have syntax as 

```SQL
SHOW ALL -- Postgres
SHOW VARIABLES -- MySQL, Snowflake, Single Store
SHOW PARAMETERS -- Oracle
exec sp_configure -- MSSQL
```

Reply:
done, it is called `show parameters` now